ui/components/app/transaction-detail/transaction-detail.component.js

```diff
@@ -11,21 +11,28 @@ import { COLORS } from '../../../helpers/constants/design-system';
 import { PRIORITY_LEVEL_ICON_MAP } from '../../../helpers/constants/gas';
 import { useI18nContext } from '../../../hooks/useI18nContext';
 
-export default function TransactionDetail({ rows = [], onEdit }) {
+export default function TransactionDetail({
+  rows = [],
+  onEdit,
+  userAcknowledgedGasMissing,
```

Comment:
Should we set a default value here?  I know undefined is falsy, but maybe we should get into this habit.  I'm all ears from others on this.

Reply:
Parent component will always pass a boolean, I should make it a required value rather.

Reply:
Also fixed